### PR TITLE
Fixes error visibility

### PR DESCRIPTION
**Pull request by JosephSalisbury**

### CODE REVIEW

File: error.go [hunk 1]
```diff
@@ -4,11 +4,11 @@ import (
 	"github.com/giantswarm/microerror"
 )
 
-var InvalidConfigError = &microerror.Error{
+var invalidConfigError = &microerror.Error{
 	Kind: "InvalidConfigError",
```
Comment (xh3b4sd) [inline error.go]: Same here.

**Comment by JosephSalisbury** [comment]
@xh3b4sd ptalaaaaaaa

**Comment by JosephSalisbury** [comment]
fucking examples. fixing

**Comment by JosephSalisbury** [comment]
here we go folks

**Comment by JosephSalisbury** [comment]
so aligned

**Pull request merged**